### DISCUSSION TITLE
Cleaner map: Additional icon placement option on nodes

### DISCUSSION THREAD

**Discussion by theelectricsoul** [freeplane/freeplane#934] [Ideas]
Hello

## Icons help but are a little messy :)
Clean maps are nice. Especially for people whom you present the maps to. Icons are also nice, as they help to understand context fast. but they mess up the map. Love and hate ;)
.
.
## Text is not aligned with icons in play🔴
The nodes itself are aligned, but not the text. This makes it hard on the brain :) 

![image](https://user-images.githubusercontent.com/10260832/211518851-fa9969a0-bf69-4655-832a-f91fd63f8ca0.png)
.
.
## How about a additional option for placement :) 🟢
![image](https://user-images.githubusercontent.com/10260832/211518380-58e1bdc8-a17d-461f-829b-37c763c725d7.png)

**Comment by dpolivaev** [comment]
What if the node shape is not a bubble, but a fork or a hexagon?

**Reply by theelectricsoul** [reply]
IMHO
I'd also say, that in general icons directly in front of the text are hard on the brain. Icons are additional information that the viewer is not interested in primarily. The brain has to seperate the icons from the text before the text can be consumed.

**Reply by dpolivaev** [reply]
I know how it is now. with the fork and the hexagon. I do not understand which new placement you suggest for them.

**Reply by theelectricsoul** [reply]
![freeplane Node design drawio(2)](https://user-images.githubusercontent.com/10260832/215762902-23157eb1-bd61-4190-b4c6-cb22fb494ffc.png)

[Original mockup file for www.draw.io](https://github.com/freeplane/freeplane/files/10546416/freeplane.Node.design.1.drawio.zip)

**Comment by euu2021** [comment]
[Related discussion](https://sourceforge.net/p/freeplane/discussion/758437/thread/66dd19d25a/).
